test/pytest/receiver/test_asapo_transfer.py

```diff
@@ -157,6 +157,23 @@ def test_path_parsing(asapo_transfer, file_list, hidra_metadata):
         assert data_source == 'test001'
 
 
+def test_path_parse_offset(asapo_transfer):
+    ragexp_template = "current/raw/(?P<scan_id>.*)_(?P<file_idx_offset>.*)_(?P<file_idx_in_scan>.*).h5"
+    asapo_transfer.asapo_worker.file_regex = ragexp_template
+
+    file_path = "current/raw/data/tilt_27_048_041_data_070421_000001.h5"
+    data_source, stream, file_idx = asapo_transfer.asapo_worker._parse_file_name(file_path)
+    assert file_idx == 70422
+
+    file_path = "current/raw/data/tilt_27_048_041_data_070421.h5"
+    data_source, stream, file_idx = asapo_transfer.asapo_worker._parse_file_name(file_path)
+    assert file_idx == 70421
```

Comment:
Usually one would write separate tests with descriptive names, e.g., the first could be called as is and the second could be called `test_path_parse_offset_not_an_int`.